darts/ad/scorers.py

```diff
@@ -0,0 +1,1582 @@
+                list_series_1[idx], _ = self._check_probabilistic_case(series)
+        else:
+            for idx, (s1, s2) in enumerate(zip(list_series_1, list_series_2)):
+                _sanity_check_2series(s1, s2)
+                list_series_1[idx], _ = self._check_probabilistic_case(s1)
+                list_series_2[idx], _ = self._check_probabilistic_case(s2)
+
+        self._fit_core(list_series_1, list_series_2)
+
+    def _diff_sequence(
+        self, list_series_1: Sequence[TimeSeries], list_series_2: Sequence[TimeSeries]
+    ) -> Sequence[TimeSeries]:
+        """Calls the function _diff() on every pair (s1,s2) in the list (list_series_1,list_series_2).
+
+        list_series_1 and list_series_2 must have the same length n.
+        Each pair of series in list_series_1 and list_series_2 must be of the same length L and width/dimension W.
+
+        Parameters
+        ----------
+        series_1
+            1st sequence of time series
+        series_2:
+            2nd sequence of time series
+
+        Returns
+        -------
+        TimeSeries
+            Sequence of series of length n
+        """
+        list_series = []
+
+        for (series1, series2) in zip(list_series_1, list_series_2):
+            list_series.append(self._diff(series1, series2))
+
+        return list_series
+
+    def _diff(self, series_1: TimeSeries, series_2: TimeSeries) -> TimeSeries:
+        """Applies the reduced_function to the two time series. Converts two time series into 1.
+
+        series_1 and series_2 must be of the same length L and width/dimension W.
+
+        Parameters
+        ----------
+        series_1
+            1st time series
+        series_2:
+            2nd time series
+
+        Returns
+        -------
+        TimeSeries
+            series of length L and width/dimension W.
+        """
+        series_1, series_2 = _return_intersect(series_1, series_2)
+
+        if self.reduced_function == "abs_diff":
+            return (series_1 - series_2).map(lambda x: np.abs(x))
+        elif self.reduced_function == "diff":
+            return series_1 - series_2
+        else:
+            # found an non-existent reduced_function
+            raise ValueError(
+                f"Metric should be 'diff' or 'abs_diff', found {self.reduced_function}"
+            )
+
+
+# FittableAnomalyScorer
+
+
+class GaussianMixtureScorer(FittableAnomalyScorer):
+    """GaussianMixtureScorer anomaly score
+
+    Wrapped around the GaussianMixture scikit-learn function.
+    Source code: <https://github.com/scikit-learn/scikit-learn/blob/f3f51f9b6/sklearn/mixture/_gaussian_mixture.py#L456>
+    """
+
+    def __init__(
+        self,
+        window: Optional[int] = None,
+        n_components: int = 1,
+        reduced_function=None,
+        component_wise: bool = False,
+    ) -> None:
+
+        """
+        A Gaussian mixture model is trained on the training data when the ``fit()`` method is called.
```

Comment:
I find these kinds of comments not very clear. What is the "training data" and what are the "samples" that the log-likelihood is computed on? I think you could rephrase, to explain how exactly the GMM is trained when `fit(actual_series, pred_series)` and `score(actual_series, pred_series)` are called. I think the comment you have below (detailing training) is much more clear. 

If you want to give a gist you could e.g. write something along the lines:
```
Upon calling fit(actual_series, pred_series), the difference between actuals and predictions is obtained, then a moving window is applied, which results in a set of vectors of size W, where W is the window size. The GMM model is trained on these vectors.
```